romancal/flatfield/flat_field.py

```diff
@@ -109,21 +111,21 @@ def apply_flat_field(science, flat):
     flat_data[np.where(flat_bad)] = 1.0
     # Now let's apply the correction to science data and error arrays.  Rely
     # on array broadcasting to handle the cubes
-    science.data /= flat_data
+    science.data = u.Quantity((science.data.value / flat_data), ru.electron / u.s, dtype=science.data.dtype)
```

Comment:
What units does science.data start with  We already forced it to have e/s in the schema?  flat_data is probably dimensionless, so I would have guessed that science.data /= flat_data would just work?  Am I missing something?